app/views/candidate_interface/references/type/new.html.erb

```diff
@@ -12,3 +12,12 @@
     <% end %>
   </div>
 </div>
+
+<%= render(
+    CandidateInterface::ApplicationFeedbackComponent.new(
+      section: 'application_references',
+      path: 'candidate_interface_references_type_path',
+      page_title: t('page_titles.references_type')
+    )
+  )
+%>
```

Comment:
Could we add this to the layout instead and only have it render on a whitelist of pages? Would save us some duplication.